src/converse-dragresize.js

```diff
@@ -143,6 +159,81 @@ converse.plugins.add('converse-dragresize', {
             'allow_dragresize': true,
         });
 
+        const resizeOccupants = {
+            renderDragResizeHandles (view) {
+                const flyout = view.el;
+                const div = document.createElement('div');
+                div.innerHTML = '<div class="dragresize dragresize-roster-left"></div>';
+                flyout.insertBefore(
+                    div,
+                    flyout.firstChild
+                );
+            },
+
+            onStartHorizontalResizeRoster (ev) {
```

Comment:
Don't refer to the occupants list as a roster, in XMPP roster means something very specific and we want to avoid confusion. The same applies to the `dragresize-roster-left` class above.